virttest/utils_v2v.py

```diff
@@ -950,7 +951,11 @@ def _v2v_pre_cmd():
         create virsh instance, etc.
         """
         # Cannot get mac address in 'ova', 'libvirtxml', etc.
-        if input_mode not in ['disk', 'libvirtxml', 'local', 'ova']:
+        if input_mode not in [
+            'disk',
+            'libvirtxml',
+            'local',
+                'ova'] and not skip_virsh_pre_conn:
```

Comment:
Please fix the aligning issue

Reply:
This is not an aligning issue. The format is created by autopep8. 
I think this line is intent to distinct with following line.